data/docs/userguide/logs_quick_filters.mdx

```diff
@@ -0,0 +1,82 @@
+---
+date: 2024-12-02
+title: Logs Quick Filters
+id: logs_quick_filters
+---
+
+The Logs Explorer comes with quick filters which allows filtering of logs based on key attributes making it easier to analyze and debug applications.
+Here's a detailed explaination of the available filters and how to use them -
+
+<figure data-zoomable align="center">
+  <img
+    src="/img/docs/logs-management/ui/logs-quick-filters.webp"
+    alt="Quick Filters in Logs Explorer"
+  />
+  <figcaption>
+    <i> Quick Filters in Logs Explorer </i>
+  </figcaption>
+</figure>
+
+### Available Filters
+
+1. **Severity Text**:
+   - **Description**: Filter logs based on their severity text.
+   - **Usage**: Useful for quickly filtering in FATAL logs when some incident occurs or filter in DEBUG logs when debugging some issue.
+   - **OTEL Attribute**: Maps to `severity_text` opentelemetry attribute.
```

Comment:
```suggestion
   - **OTel Attribute**: Maps to [`severity_text`](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitytext) opentelemetry attribute.
```
https://opentelemetry.io/docs/contributing/style-guide/#opentelemetryio-word-list

Please confirm if this hyperlink is correct - https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitytext